src/drivers/cmakeDriver.ts

```diff
@@ -1603,7 +1603,7 @@ export abstract class CMakeDriver implements vscode.Disposable {
      * The currently running process. We keep a handle on it so we can stop it
      * upon user request
      */
-    private _currentBuildProcess: proc.Subprocess | null = null;
+    protected _currentBuildOrConfigureProcess: proc.Subprocess | null = null;
```

Comment:
please rename `_currentBuildOrConfigureProcess` to `currentBuildOrConfigureProcess`.